version/version.go

```diff
@@ -1,4 +1,4 @@
 package version
 
 // VERSION ...
-const VERSION = "1.46.0"
+const VERSION = "1.47.0"
```

Comment:
Forgot to mention sorry that the version we will need is 1.48.0, because we will need to release a new CLI version and the currently released version is 1.47.0